config/_default/hugo.toml

```diff
@@ -239,7 +244,7 @@ hide_summary = false
 
 [params.footer.recent_posts]
 enable = false
-
+        
```

Comment:
this seems wrong

Reply:
I am spoiled by rustfmt. Fixed, and turned on trailing whitespace viz in VSCode :)